app/models/category.php

```diff
@@ -0,0 +1,25 @@
+<?php
+class Category extends AppModel
+{
+    const TABLE_NAME = 'category';
+
+    public static function getAll()
+    {
+        $db = DB::conn();
+        $rows = $db->rows(sprintf('SELECT * FROM %s', self::TABLE_NAME));
```

Comment:
You can simply type the string po rather than using a function to generate the string. If you're considering micro-optimizations :smile: